perf/lua/column_scan_module.c

```diff
@@ -17,6 +18,14 @@
 # define ENABLE_ARROW 1
 #endif /* ENABLE_MEMCS_ENGINE || ENABLE_QUIVER_ENGINE */
 
+static int
+init_lua_func(struct lua_State *L)
+{
+	(void)L;
+	srand(time(NULL));
```

Comment:
This could as well be done in Lua. No need to introduce a Lua module function for this. It'd also be nice to print the random seed to the log.

Reply:
Changed to:
```lua
local seed = os.time()
log.info('Random seed: ' .. seed)
math.randomseed(seed)
```